servers/src/main/java/tachyon/master/file/journal/AddMountPointEntry.java

```diff
@@ -25,20 +25,21 @@
 import tachyon.master.journal.JournalEntryType;
 
 /**
- * This class represents a journal entry for recording the addition of a new mount point.
+ * This class represents a journal entry for addition of a new mount point.
  */
 public class AddMountPointEntry extends JournalEntry {
   private final String mTachyonPath;
   private final String mUfsPath;
 
   /**
-   * Creates a new instance of <code>AddMountPointEntry</code>.
+   * Creates a new instance of {@link AddMountPointEntry}.
```

Comment:
I try to use link any time I refer to a Class or a Class member irrespective of whether it is a current class or not; hyperlinks FTW.